_posts/2023-11-23-announcing-dp-social-inbox.md

```diff
@@ -0,0 +1,101 @@
+
+Through our existing suite of tools, we already make it possible to publish static [Jekyll](https://jekyllrb.com/) websites to the dweb. With static websites, code and content are delivered to the browser as simple HTML/CSS files, without having to access an external server to load content. The benefit of static websites is that they load faster, do not use plug-ins that require constant updates, and tend to be more secure overall. We realized that we could lean on these features, while incorporating social interactions into websites using ActivityPub. 
+
+So over the course of this year, we built the **Social Inbox**, which allows people to integrate their website with the Fediverse and engage with readers there. [You can read more about the technical details of how the Social Inbox works here](https://blog.mauve.moe/posts/distributed-press-social-inbox#how-the-inbox-works). 
+
+With the Social Inbox enabled on your site, you can:
+
+* **Post to the Fediverse**: New articles can be posted to the Fediverse where followers can read, re-post, favorite, and send replies.
+
+* **Approve replies as comments**: When posts receive replies on the Fediverse, the website owner can approve them as reader comments that are then posted alongside the original article on the website.
+
+* **Streamline moderation**: Approve messages and follow requests one-by-one, automate the process on your own, or subscribe to existing block lists. (For now, Distributed.Press subscribes to [GardenFence](https://github.com/gardenfence/blocklist/tree/main) to automatically block servers that have shown to be sources of hateful rhetoric, harassment, and spam.) 
+
+* **Download your follower list**: If you would rather move your account to a different server, you can switch your Social Inbox instance very easily without needing to change the domain name for your site.
+
+To see which Fediverse clients are compatible with the Social Inbox, you can follow this [Github issue](https://github.com/hyphacoop/social.distributed.press/issues/24).
+
+## Sutty Integration
+
+If you aren't able or interested in coding your way to creating a static website or implementing the Social Inbox yourself, we have also integrated it into [Sutty](https://sutty.nl/en), a content management system for static Jekyll websites. All you need to do is set up a website on their platform and enable the Social Inbox in the configuration panel. Then, you will see a panel that will allow you to approve/deny replies as comments to your posts.
+
+All of Sutty's templates are compatible with the Social Inbox, in addition to the Magazine template, which is based on the layout of COMPOST magazine, our sister project. For a step-by-step guide on how to use the Social Inbox with Sutty, check out this [LINK:demo video].
+
+[LINK WHEN AVAILABLE: You can see how it looks by checking out our account on the Fediverse: @HyphaCoop@hypha.coop]
```

Comment:
How do we link to this account?

Reply:
That's a good question. I also wonder if we should find a way to rename this profile as "Hypha Dripline" or "The Dripline from Hypha" or something like that?